### ISSUE TITLE
Unable to click inside notification boxes

### ISSUE THREAD

**Issue by theiliad** [carbon-design-system/carbon-website#655]
I believe this issue is coming from `carbon-components`. Not sure if the website is adding any styles to `carbon-components` so it also be an issue with the styles in this repo.

Created https://github.com/carbon-design-system/carbon/pull/4838

### Steps to reproduce
1. Navigate to https://www.carbondesignsystem.com/data-visualization/basic-charts
2. Click on the links inside this box or try highlighting the text
![image](https://user-images.githubusercontent.com/14989804/70451529-86011300-1a73-11ea-9dc3-271f98e661ef.png)

**Comment by theiliad** [comment]
This is getting addressed by @jendowns is doing over in https://github.com/carbon-design-system/carbon/pull/4805

**Comment by alisonjoseph** [comment]
This will be fixed upstream in Carbon
